src/pages/host/getHostEventString.tsx

```diff
@@ -35,20 +35,36 @@ export const getHostEventString = (
 ) => {
   switch (eventType) {
     case HostEvent.Created:
-      return `Host created`;
+      return <span data-cy="created">Host created</span>;
     case HostEvent.AgentDeployFailed:
-      return `New agent deploy failed`;
+      return <span data-cy="agent-deploy-failed">New agent deploy failed</span>;
     case HostEvent.ProvisionError:
-      return `Host encountered error during provisioning`;
+      return (
+        <span data-cy="provision-error">
+          Host encountered error during provisioning
+        </span>
+      );
     case HostEvent.Started:
-      return `Host start attempt ${data.successful ? "succeeded" : ""}`;
+      return (
+        <span data-cy="started">
+          Host start attempt {data.successful ? "succeeded" : ""}
```

Comment:
I realize this was code existed before this diff but it is a good practice to save repeated copy in a variable to allow easier updates